champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/Skill.java

```diff
@@ -73,12 +112,18 @@ public final void loadConfig() {
         maxLevel = getConfig("maxlevel", 5, Integer.class);
 
         if (this instanceof CooldownSkill) {
-            cooldown = getConfig("cooldown", 0, Integer.class);
+            cooldown = getConfig("cooldown", 1.0, Double.class);
+            cooldownDecreasePerLevel = getConfig("cooldownDecreasePerLevel", 1.0, Double.class);
         }
 
         if (this instanceof EnergySkill) {
             energy = getConfig("energy", 0, Integer.class);
         }
+        canUseWhileSlowed = getConfig("canUseWhileSlowed", true, Boolean.class);
+        canUseWhileSilenced = getConfig("canUseWhileSilenced", false, Boolean.class);
+        canUseWhileStunned = getConfig("canUseWhileStunned", false, Boolean.class);
+        canUseWhileLevitating = getConfig("canUseWhileLevitating", false, Boolean.class);
+        canUseInLiquid = getConfig("canUseInLiquid", false, Boolean.class);
```

Comment:
Perhaps we should only check this if the skill is NOT a passive or global skill? Might keep the config a little cleaner

Reply:
Is this something that is consistent among all skills of a type? I understand the problem, I don't like how cluttered the config is.

A better solution might be to check if that config option is present for that skill, if not, use the default.

Reply:
These limitations are only applied to Sword / Axe / Bow / Drop item skills (which are a passive I guess)

That would also be a viable solution to keep things a little cleaner